app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "workshop.portal@gmail.com"
+  default from: Rails.configuration.default_reply_to_email
```

Comment:
This change is wrong. We actually send emails with this adress.
I agree that it might be good to read this address from the config/environment too, but from and reply-to are two different emails.
I dont know the correct var atm, I'll have a look later on